vendor/assets/stylesheets/uswds_overrides/components/_footer.scss

```diff
@@ -1,15 +1,143 @@
-.usa-footer-big {
-  .usa-footer-primary-link {
-    background-image: image-url('uswds/arrow-down.png');
-    background-image: image-url('uswds/arrow-down.svg');
+.site-footer{
+  html, body , div, span, applet, object, iframe,
+  h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+  a, abbr, acronym, address, big, cite, code,
+  del, dfn, em, img, ins, kbd, q, s, samp,
+  small, strike, strong, sub, sup, tt, var,
+  b, u, i,
+  dl, dt, dd, ol, ul, li,
+  fieldset, form, label, legend,
+  table, caption, tbody, tfoot, thead, tr, th, td,
+  article, aside, canvas, details, embed,
+  figure, figcaption, footer, header, menu, nav,
+  output, ruby, section, summary,
+  time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
   }
 
-  ul {
-    &.hidden {
-      .usa-footer-primary-link {
-        background-image: image-url('uswds/arrow-right.png');
-        background-image: image-url('uswds/arrow-right.svg');
-      }
-    }
+  * { box-sizing: border-box; }
+  *:before, *:after { display: none; }
+
+  font-family: Source Sans Pro, Verdana, sans-serif;
+  font-size: 18px;
```

Comment:
Absolutely! I can the `base-font-size` to 18px (to match the prototype), but that might break other things across the site.